documentation/modules/ROOT/pages/02-architecture.adoc

```diff
@@ -1,104 +1,255 @@
+
+The mesh's inherent resilience is further bolstered by its exponential backoff strategy, which it employs when encountering operational failures.
+This mechanism ensures that the system retries the operation until it succeeds, thus mitigating the risk of data loss or system disruption due to transient issues.
+
+By integrating the _Event Mesh_ into the system architecture, several architectural benefits are achieved:
+
+* **Decomposition of the application into independently functioning services**:
+This approach facilitates a division of labor, with each service handling specific responsibilities.
+This not only enhances maintainability but also fosters scalability, as services can be independently scaled based on their demands.
 
-=== Using images 
+* **Improved business alignment**:
+By embracing an eventual consistency model, the _Event Mesh_ aligns closely with the inherent nature of most real-world business processes.
+Unlike traditional transactional systems that strive for immediate, irreversible consistency, the _Event Mesh_ allows for a more flexible and adaptive approach to data consistency.
+This results in better alignment with business requirements, as it supports scenarios where multiple services collaborate and synchronize their operations, making the whole state eventually consistent, without the constraint of strict, synchronous consistency.
 
-image::red_hat_open-hybrid-cloud.png[width=30%]
+* **Avoidance of systemic issues**:
+The _Event Mesh_'s error-handling mechanism, through retries and event persistence, aids in minimizing the impact of failures on the end user.
+This is crucial as it prevents bugs and outages from becoming visible to the user, thereby preserving the system's perceived responsiveness.
 
-{empty}
+* **Enhanced system performance**:
+The system becomes more responsive, as the end user no longer needs to wait for multiple, often independent, operations to complete successfully.
+The _Event Mesh_'s event-driven model, coupled with the retries and event persistence, ensures that critical state changes are propagated swiftly and reliably, thereby improving the overall user experience.
 
-=== Embedding HTML
+=== _Event Mesh_ Flow
 
-++++
-  <br>
-  <h3> Embed HTML by surrounding it with with four +s before and after. </h3>
-  <span>View the ascii doc to learn more</span>
-  <br> 
-++++
+The event-driven flow enables eventual consistent collaboration and state synchronization between services, fostering a resilient, scalable, and developer-friendly system architecture.
 
+A usual flow may look like:
 
-=== Different decorators
+1. An end-user application sends a request, which forms a _Command_ type event, which is transferred to the _Event Mesh_.
+2. The _Event Mesh_ (Broker) persists the event in a queue (e.g. Kafka).
```

Comment:
```suggestion
2. The _Event Mesh_ (Broker) persists the event in a queue (like a Kafka topic, but the implementation is hidden from the user).
```

Reply:
I'd call it Apache Kafka (legal aspect from the ASF)